packages/dbos-cloud/.npmignore

```diff
@@ -0,0 +1,12 @@
+# ignore all TS files
```

Comment:
Should we add an `.npmignore` file for other packages in this repo? For example, we probably don't want `dbos-config.schema.json` or `dbos-test-config.yaml` in the SDK package

Reply:
Updated the base repo `.npmignore`